pyproject.toml

```diff
@@ -112,7 +112,16 @@ include = [
 ]
 
 [tool.pdm]
-ignore_package_warnings = ["sphinx", "slotscheck"]
+ignore_package_warnings = [
+  "alabaster",
+  "sphinxcontrib-*",
+  "sphinx-*",
+  "sphinx",
+  "pre-commit",
+  "pydata-sphinx-*",
+  "slotscheck",
+  "autobahn","accessible-pygments"
```

Comment:
My console is a bit noisy without these suppressions

Reply:
After running `make lint`?